app/views/users/show.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1> Hello views </h1>
-<p> this is a test</p>
\ No newline at end of file
+<div class="user">
+<%= render 'partials/user', user: @user %>
+<div class='bio'>
+<p><%= @user.bio %></p>
+</div>
+<ul class='card'>
+    <% @user.recent_three_posts.each do |post| %>
+      <a href="/users/<%= @user.id %>/posts/<%= post.id %> ">
+        <%= render 'partials/post', post: post %>
+      </a>
+    <% end %>
+</ul>
+
+  <%= link_to "See all posts", user_posts_path(@user.id)%>
+</div>
+
```

Comment:
- [ ] Your layout should match the wireframe given to you. The list of posts should be somewhat centered horizontally and the "See all posts" button should be centered and at the bottom of the page. Kindly do that.

  See [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAABY4RVEM2M5SSQDF3WYVDHTIY4KOBTQ)

  ![utitw](https://user-images.githubusercontent.com/54848363/205382148-ae6e6b3d-e757-4b3f-814b-cec95fe7a51d.jpg)